### ISSUE TITLE
Turning off theme toggle breaks collapsible callouts

### ISSUE THREAD

**Issue by elkalt** [KosmosisDire/obsidian-webpage-export#176]
Not including the theme toggle causes a TypeError on line 501 of `webpage.js`:
```
Uncaught TypeError: Cannot set properties of null (setting 'checked')
    at setThemeToggle (webpage.js:501:18)
```

```javascript
function setThemeToggle(state, instant = false) {
  let toggle = document.querySelector(".theme-toggle-input");
  toggle.checked = state;  // TypeError is here
  ....
```

This breaks in particular collapsible callouts (they no longer open or close) but may affect other things as well.  Probably just need to extend the null check if statement in `setupThemeToggle()` to include the code that sets the initial toggle state based on the body theme class.

**Comment by KosmosisDire** [comment]
Thanks for reporting this, I will try to reproduce it when I have time

**Comment by KosmosisDire** [comment]
This should be fixed in v1.7.0! Let me know if it is not
